whisper/whisperv6/peer_test.go

```diff
@@ -101,7 +101,7 @@ var debugMode = false
 var prevTime time.Time
 var cntPrev int
 
-func XTestSimulation(t *testing.T) {
+func TestSimulation(t *testing.T) {
```

Comment:
Same here, rationale for re-enabling this test?

Reply:
See comment on PR - these tests are not related to Swarm, and I don't see them failing on go-ethereum master.